### ISSUE TITLE
Some SC tracks throw a 401 on 1.3.29

### ISSUE THREAD

**Issue by aikaterna** [sedmelluq/lavaplayer#255]
I am a Lavalink user, but my jar has Lavaplayer 1.3.29: https://github.com/sedmelluq/lavaplayer/commit/cc6d29cf5b642b985f9db157d40a4134cf04f988 This also seems to be present in 1.3.27, also tried this on a JDA bot.

I'm receiving a 401 error for some tracks, but not others. 

Playlist url: https://soundcloud.com/aikaterna/sets/p5-1

Doesn't work - https://soundcloud.com/aikaterna/029a
Works - https://soundcloud.com/aikaterna/081a
Works - https://soundcloud.com/aikaterna/072a

Playlist url: https://soundcloud.com/daniel-galabov-769051185/sets/lofi-triphop-1

Many tracks in this playlist throw the error, possibly about the first 10-20 or more, I stopped my testing on it after a bunch errored out

Playlist url: https://soundcloud.com/lupinthebeatsmith/sets/boom-bap-playlist

All seem to work

```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43) ~[lavaplayer-1.3.29.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:103) ~[lavaplayer-1.3.29.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349) [lavaplayer-1.3.29.jar!/:na]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_232]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_232]
        at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_232]
Caused by: java.io.IOException: Invalid status code for soundcloud stream: 401
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioTrack.loadFromMp3Url(SoundCloudAudioTrack.java:70) ~[lavaplayer-1.3.29.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioTrack.process(SoundCloudAudioTrack.java:43) ~[lavaplayer-1.3.29.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:91) ~[lavaplayer-1.3.29.jar!/:na]
        ... 4 common frames omitted
```

**Comment by sedmelluq** [comment]
Should be fixed in `1.3.31.`

**Comment by aikaterna** [comment]
Thank you, 1.3.31 or later resolves this issue.
